src/app.ts

```diff
@@ -29,7 +30,7 @@ routerOpen.get("/api/version", (ctx) => {
 
 routerOpen.post("/api/auth/login", authLogin);
 routerOpen.post("/api/user", userPost);
-
+routerOpen.post("/api/user/:id", userUpdate);
```

Comment:
routerOpen are routes that do not need and do not pass auth validation, use the `userRoutes` component: https://github.com/biantris/restris/blob/main/src/routes/userRoutes.ts#L15